identity/did/bootstrapper.go

```diff
@@ -80,7 +80,7 @@ func getAnchorAddress() common.Address {
 // ---------------------------------------------------------------------------------------------------------------------
 func migrateNewIdentityContracts() {
```

Comment:
This logic in this file is only for testing, we should think about removing it to the test bootstapper instead.